CMS-Enterprise/easi-app
# [EASI-4598] GRB reviewer email notifications

# EASI-4325 EASI-4598

## Description

- Adds logic for handling emails around the GRB Review start date.
- Fixes email script sender not sending emails with only CC/BCC addresses.
- Adds functional helpers for slices.

## How to test this change

- Start/Seed app and visit [this intake](http://localhost:3000/it-governance/5af245bc-fc54-4677-bab1-1b3e798bb43c/grb-review)
- Open [mailcatcher](http://localhost:1080/)
- Add a reviewer, no emails.
- Start the review, see email go out to current reviewers. (Note: Mailcatcher files all addresses under the "To")
- Add a reviewer, that reviewer should receive an individual email.
- Fire off this mutation to see error when trying to start a review after one has already been started:

```gql
mutation startReview {
  startGRBReview(input: {
  	systemIntakeID: "5af245bc-fc54-4677-bab1-1b3e798bb43c"
  })
}
```

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
